### PR TITLE
BugFix On-CatalogueComp Table Format Change FilterDetails on ResultPDF

### PR DESCRIPTION
**Pull request by DeepikaDK001**

### CODE REVIEW

File: src/NHSD.GPIT.BuyingCatalogue.WebApp/Views/CompetitionResultsPdf/Index.cshtml [hunk 1]
```diff
@@ -227,10 +225,9 @@
                             <nhs-summary-list-row label-text="@nonPriceElementWeighting.Key.EnumMemberName()">@nonPriceElementWeighting.Value%</nhs-summary-list-row>
                         }
                     </nhs-summary-list>
```
Comment (dssherlock) [inline src/NHSD.GPIT.BuyingCatalogue.WebApp/Views/CompetitionResultsPdf/Index.cshtml:227]: This style appears to be re-used a lot. Maybe worth putting in the css file as a class?

File: src/NHSD.GPIT.BuyingCatalogue.WebApp/Views/CompetitionResultsPdf/Index.cshtml [hunk 2]
```diff
@@ -206,9 +206,7 @@
                 }
 
                 <h2>Filter details</h2>
```
Comment (dssherlock) [inline src/NHSD.GPIT.BuyingCatalogue.WebApp/Views/CompetitionResultsPdf/Index.cshtml:208]: Has the new FilterDetails partial replaced the Partials/FilterDetailsPartial one? If so, the old one should be deleted.

File: src/NHSD.GPIT.BuyingCatalogue.WebApp/Views/CompetitionResultsPdf/_FilterDetails.cshtml [hunk 3]
```diff
@@ -0,0 +1,183 @@
...
+                }
+            }
+        </nhs-summary-list-row>
+        <nhs-summary-list-row label-text="Interoperability">
+            @if (Model.HasInteroperabilityIntegrationTypes())
+            {
+                @for (int j = 0; j < Model.FilterDetails.InteropIntegrationTypes.Count(); j++)
+                {
+                    @if (@Model.FilterDetails.InteropIntegrationTypes[j].EnumMemberName() == "IM1")
+                    {
```
Comment (dssherlock) [inline src/NHSD.GPIT.BuyingCatalogue.WebApp/Views/CompetitionResultsPdf/_FilterDetails.cshtml:151]: This line can be outside the if statement as it appears in both Integration Types

**Review by dssherlock — APPROVED** [review]

**Review by imranbashir01 — APPROVED** [review]

File: src/NHSD.GPIT.BuyingCatalogue.WebApp/Views/CompetitionResultsPdf/Index.cshtml (hunk 2 above)
Comment (DeepikaDK001) [reply src/NHSD.GPIT.BuyingCatalogue.WebApp/Views/CompetitionResultsPdf/Index.cshtml:208]: this new partial has been created for PDF table styling and has not been replaced.

**Pull request merged**